script.js

```diff
@@ -0,0 +1,99 @@
+let teacherInfo = [
+    { name: 'Ahmadreza Azad', skills: 'Bachelor of computer sience and English language', details: 'Teach basic English skills including reading, writing, and speaking.', img: 'assets/images/teacher-1.jpeg' },
```

Comment:
- [optioanal] please kindly increase the image and font size of the speakers so it can be more user friendly. the seem too small compared to the original design.
![Screenshot (880)](https://user-images.githubusercontent.com/65298048/225625815-f4616e9c-a895-437e-aa9e-85e593b29840.png)